kurma-earthworks-v1-stage/solrconfig.xml

```diff
@@ -38,7 +38,18 @@
     </autoCommit>
   </updateHandler>
 
+  <updateProcessor class="solr.RemoveBlankFieldUpdateProcessorFactory" name="remove_blanks" />
 
+  <updateRequestProcessorChain name="earthworks-processor" processor="remove_blanks">
+    <processor class="solr.LogUpdateProcessorFactory" />
+    <processor class="solr.RunUpdateProcessorFactory" />
+  </updateRequestProcessorChain>
+
+  <requestHandler name="/update" class="solr.UpdateRequestHandler">
```

Comment:
This didn't exist already (as it does locally in the EW codebase).  Hopefully this is 👌 to add w/o any other concerns.